app/routes/_index.tsx

```diff
@@ -1,7 +1,7 @@
 import type { MetaFunction } from '@remix-run/node'
 import hero from '~/assets/hero.png'
+import Logo from '~/components/Logo'
 import { Button } from '~/components/ui/button'
```

Comment:
**nitpick**: The import path for the Logo component was changed. Ensure this is intentional and consistent across the project.